src/providers/seventv/SeventvEmotes.cpp

```diff
@@ -131,114 +139,44 @@ boost::optional<EmotePtr> SeventvEmotes::emote(const EmoteName &name) const
-                id
-                display_name
-                login
-                twitch_id
-            }
-        }
-    })";
-
-    variables.insert("query", QString());
-    variables.insert("globalState", "only");
-    variables.insert("page", 1);  // TODO(zneix): Add support for pagination
-    variables.insert("limit", maxPageSize);
-    variables.insert("pageSize", maxPageSize);
+    qCDebug(chatterinoSeventv) << "[7TVEmotes] Loading 7TV Global Emotes";
 
-    payload.insert("query", query.replace(whitespaceRegex, " "));
-    payload.insert("variables", variables);
+    QString url("https://api.7tv.app/v2/emotes/global");
 
-    NetworkRequest(apiUrlGQL, NetworkRequestType::Post)
+    NetworkRequest(url)
         .timeout(30000)
-        .header("Content-Type", "application/json")
-        .payload(QJsonDocument(payload).toJson(QJsonDocument::Compact))
-        .onSuccess([this](NetworkResult result) -> Outcome {
-            QJsonArray parsedEmotes = result.parseJson()
-                                          .value("data")
-                                          .toObject()
-                                          .value("search_emotes")
-                                          .toArray();
-            qCDebug(chatterinoSeventv)
-                << "7TV Global Emotes" << parsedEmotes.size();
-
-            auto pair = parseGlobalEmotes(parsedEmotes, *this->global_.get());
+        .onSuccess([this](auto result) -> Outcome {
+            auto emotes = this->emotes();
+            auto pair = parseGlobalEmotes(result.parseJsonArray(), *emotes);
             if (pair.first)
                 this->global_.set(
                     std::make_shared<EmoteMap>(std::move(pair.second)));
             return pair.first;
+            return Failure;
         })
         .execute();
 }
 
 void SeventvEmotes::loadChannel(std::weak_ptr<Channel> channel,
                                 const QString &channelId,
+                                const QString &channelDisplayName,
                                 std::function<void(EmoteMap &&)> callback,
                                 bool manualRefresh)
 {
-    qCDebug(chatterinoSeventv)
-        << "Reloading 7TV Channel Emotes" << channelId << manualRefresh;
-
-    QJsonObject payload, variables;
-
-    QString query = R"(
-        query loadUserEmotes($login: String!) {
-            user(id: $login) {
-                emotes {
-                    id
-                    name
-                    provider
-                    provider_id
-                    visibility
-                    mime
-                    owner {
-                        id
-                        display_name
-                        login
-                        twitch_id
-                    }
-                }
-            }
-        })";
+    qCDebug(chatterinoSeventv) << "[7TVEmotes] Reloading 7TV Channel Emotes"
+                               << channelId << manualRefresh;
```

Comment:
This is an unnecessary change - logging categories exist for a reason and are there to not hardcode categories like this.